web/src/js/routes/KnowYourRightsRoute.jsx

```diff
@@ -8,7 +8,70 @@ class KnowYourRightsRoute extends React.Component {
 
   render() {
     return (
-      <div className='KnowYourRightsRoute'>KYR goes here</div>
+      <div className='KYRRoute'>
+        <div className='RightsList'>
```

Comment:
If you feel like it, you can pull this content into the translations file. I'm also happy to do that once this is merged in.